docs/environments.md

```diff
@@ -0,0 +1,34 @@
+# Environments
+
+We have several environments where the app can be deployed:
+
+
+|Env|URL|Purpose|How it's deployed|Backend env|
+|---|---|---|---|---|
+|local|http://localhost:3000/app|local development|`yarn start`|staging|
+|PRs   |`https://pr<PR_NUMBER>--safereact.review-safe.gnosisdev.com/app`|peer review & feature QA|for all PRs on push|staging|
+|release PRs|`https://pr<PR_NUMBER>--safereact.review-safe.gnosisdev.com/app`|pre-release regression testing|in PRs to `main`|**production** (sic!)|
```

Comment:
This is the same link as above. Maybe we should label the above env as: (Release) PRs. What do you think?

Reply:
I think you're splitting hairs here. It _is_ the same link.